utils/vscode/README.md

```diff
@@ -25,8 +25,12 @@ code --install-extension out/carbon.vsix
 
 ## Development
 
-1. Open utils/vscode folder in VS Code.
-2. Launch the extension using Run command (F5).
+1. `bazel build language_server` in project root.
```

Comment:
Maybe:
```suggestion
1. `bazel build -c opt language_server` in project root.
```

Reply:
for now debug build is fine, build clang in release mode will be very slow. 

If we add more time sensitive things (like completion), we can this.